### PR TITLE
Xiluo/gpio

### PR DESCRIPTION
**Pull request by xiluohe**

### CODE REVIEW

File: src/_arch/aarch32/cpu/boot.S [hunk 1]
```diff
@@ -11,3 +11,12 @@ _start:
     mov     fp, #0
     bl      runtime_init
 hang: b hang
+
+.globl dev_barrier
+dev_barrier:
```
Comment (polygnomial) [inline src/_arch/aarch32/cpu/boot.S:16]: /praise @Akshay-Srivatsan

**Review by polygnomial — COMMENTED** [review]
Looks good, I think we should merge the test module first and then you can merge that into your branch and write the tests in `gpio.rs`

**Comment by xiluohe** [comment]
Test module merged

**Review by polygnomial — APPROVED** [review]
Lgtm!

**Pull request merged**